### PR TITLE
Rename helm mixin to helm2

### PR DESCRIPTION
**Pull request by carolynvs**
# What does this change
* Sync helm2 page after PR is merged
* Redirect helm -> helm2
* Add page for helm3

# What issue does it fix
Part of https://github.com/getporter/helm-mixin/issues/81
Waiting on https://github.com/getporter/helm-mixin/pull/83

# Notes for the reviewer
NA

# Checklist
- [ ] Unit Tests
- [x] Documentation
- [ ] Schema (porter.yaml)

### CODE REVIEW

File: docs/content/mixins/helm2.md [hunk 1]
```diff
@@ -1,39 +1,45 @@
...
-Repositories
+Add repositories
 
 ```yaml
 - helm:
     repositories:
-      bitnami:
-        url: "https://charts.bitnami.com/bitnami"
+      stable:
+        url: "https://charts.helm.sh/stable
```
Comment (iennae) [inline docs/content/mixins/helm2.md:42]: missing a "

**Comment by iennae** [comment]
Ugh, sorry. I updated the changes in the GitHub UI and it fails the DCO check when doing that. I can't see how to rebase my updates with the signoff either.

File: docs/content/mixins/helm2.md (hunk 1 above)
Comment (carolynvs) [reply docs/content/mixins/helm2.md]: Thanks!

**Comment by carolynvs** [comment]
You can run `git rebase HEAD~1 --signoff` to add your DCO signature to the last commit. I use that when I modify a PR from the web, for example accepting suggested changes from a review.

**Comment by carolynvs** [comment]
This is completed in another PR.

**Pull request closed without merging**